odoo/addons/base/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'Base',
-    'version': '1.3',
+    'version': '1.4',
```

Comment:
No. Don't change the version. It's not needed.
The base module is in version `1.3` since 2010 (f4c0dfd0f5bfb0bafb5ed4698321eb693932267b).